### PR TITLE
Add further rollup-less fcls

### PR DESCRIPTION
**Pull request by henrylay97**
As requested by @jzennamo in yesterday's RM meeting. A further 5 fcls for running g4 without rollup.

### CODE REVIEW

File: sbndcode/JobConfigurations/standard/gen/corsika/prodcorsika_proton_intime_filter_no_shower_rollup.fcl [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+physics.producers.pdfastsimintime.SimulationLabel: "ionandscintintime:priorSCE"
+physics.filters.timefilter.SimPhotonsLiteCollectionLabel: "pdfastsimintime"
+
+services.ParticleListAction.keepGenTrajectories: ["GenInTimeSorter"]
+services.ParticleListAction.keepEMShowerDaughters: true
+
+outputs.out1.SelectEvents: [ "simulate" ]
+
+process_name: CosmicsCorsikaProtonGenAndG4InTime
+
```
Comment (marcodeltutto) [inline sbndcode/JobConfigurations/standard/gen/corsika/prodcorsika_proton_intime_filter_no_shower_rollup.fcl:67]: Can this file inherit from `prodcorsika_proton_intime_filter.fcl`?
```
#include "prodcorsika_proton_intime_filter.fcl"
services.ParticleListAction.keepEMShowerDaughters: true
```
That would be easier to maintain.

Do we also need the one with SCE?

File: sbndcode/JobConfigurations/standard/gen/overlay/prodoverlay_corsika_cosmics_proton_genie_rockbox_sbnd_no_shower_rollup.fcl [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+outputs.out1.outputCommands: [ "keep *_*_*_*"
+                             , "drop *_largeantnu_*_*"
+                             , "drop *_largeantcosmic_*_*"
+                             ]
+
+# Overidde the names of the output files
+outputs.out1.fileName: "prodgenie_cosmic_rockbox_sbnd_%p-%tc.root"
+services.TFileService.fileName: "hists_prodgenie_cosmic_rockbox_sbnd_%p-%tc.root"
+
+services.ParticleListAction.keepEMShowerDaughters: true
```
Comment (marcodeltutto) [inline sbndcode/JobConfigurations/standard/gen/overlay/prodoverlay_corsika_cosmics_proton_genie_rockbox_sbnd_no_shower_rollup.fcl:39]: Can this file inherit from `prodoverlay_corsika_cosmics_proton_genie_rockbox_sbnd.fcl`?
```
#include "prodoverlay_corsika_cosmics_proton_genie_rockbox_sbnd.fcl"
services.ParticleListAction.keepEMShowerDaughters: true
```
That would be easier to maintain.

**Review by marcodeltutto — CHANGES_REQUESTED** [review]
Thanks for adding these @henrylay97! They look good, I just have a couple of comments. Maybe I am missing something but I think we can have the fcls include the previous ones, so they are easier to maintain.

**Comment by henrylay97** [comment]
Thanks @marcodeltutto you're totally right about them inheriting from the previous fcls.

As for whether we need the sce versions, I'll let @jzennamo wade in on that, this was just the list he sent me from production requests.

**Review by marcodeltutto — APPROVED** [review]
Thanks @henrylay97! Looks good. If these are the fcls that @jzennamo requested, they look good to me.

**Comment by jzennamo** [comment]
Oh man, let me check, I might have screwed up the fcls. Likely good catch @marcodeltutto  🤦

**Comment by jzennamo** [comment]
Great catch @marcodeltutto I did want `prodcorsika_proton_intime_filter_sce.fcl`... Sorry for the misdirect @henrylay97!

**Comment by henrylay97** [comment]
What about for the prodoverlay file? @jzennamo

**Comment by pgreen135** [comment]
@henrylay97 I think that wants to be the sce version too, as we are using those for all the other samples

**Comment by jzennamo** [comment]
life saver! Yes, we'll need a SCE version of `prodoverlay_corsika_cosmics_proton_genie_rockbox_sbnd.fcl` which I thought was SCE enabled

**Comment by henrylay97** [comment]
Ah I was purely going by name. I'll take a look and push these extras :)

**Comment by henrylay97** [comment]
Okay so just to update again:
- The file `prodcorsika_proton_intime_filter_no_shower_rollup.fcl` has been replaced with `prodcorsika_proton_intime_filter_sce_no_shower_rollup.fcl`
- The file `prodoverlay_corsika_cosmics_proton_genie_rockbox_sce.fcl` has been added so we can also then add `prodoverlay_corsika_cosmics_proton_genie_rockbox_sce_no_shower_rollup.fcl`

**Comment by henrylay97** [comment]
I think @jzennamo this provides the necessary fcls now

**Review by jzennamo — APPROVED** [review]
Thank you so much for pulling these together!

**Pull request merged**